lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -91,6 +92,22 @@ protected function initialize()
     /**
      * {@inheritDoc}
```

Comment:
There is no parent method here as far as I can see. Can you provide a descriptive DocBlock?